src/ledctl.c

```diff
@@ -201,12 +205,17 @@ static const int possible_params_size = sizeof(possible_params)
 
 static int listed_only;
 
+static enum output_format output = OUTPUT_STD;
+
 enum cntrl_type get_cntrl_type(const char *cntrl)
 {
 	if (strcasecmp(cntrl, "vmd") == 0)
 		return CNTRL_TYPE_VMD;
 	else if (strcasecmp(cntrl, "npem") == 0)
 		return CNTRL_TYPE_NPEM;
+	else if (strcasecmp(cntrl, "scsi") == 0) {
```

Comment:
another case here... can we use map?

Reply:
Please elaborate, are you suggesting I create a hashmap/hashtable data type, utilize  glib hashtable, or other?

Reply:
for ibpi values- controller specific mapping we created simple struct:

```
struct ibpi_value {
	int ibpi, value;
};
```

and 2 functions:
```
int get_value_for_ibpi(enum ibpi_pattern ibpi, const struct ibpi_value ibpi_values[]);
enum ibpi_pattern get_ibpi_for_value(const int value, const struct ibpi_value ibpi_values[]);
```
I would like to have something similar for mapping enum <-> string value.
That will help us to get rid of many similar `switch` or `else if` statements later. What do you think?

Reply:
> I would like to have something similar for mapping enum <-> string value.
> That will help us to get rid of many similar `switch` or `else if` statements later. What do you think?

I'll give it a shot.